lib/karafka/instrumentation/vendors/datadog/logger_listener.rb

```diff
@@ -76,7 +76,7 @@ def on_worker_processed(event)
             info "[#{job.id}] #{job_type} job for #{consumer} on #{topic} finished in #{time}ms"
 
             current_span = client.active_span
-            current_span.finish if current_span.present?
```

Comment:
We could use `current_span&.finish`.

Reply:
We could just `client.active_span&.finish` because it's not used anywhere else.

Reply:
True, that is cleaner.

Reply:
@vitellochris would you mind updating it that way? Then we should be good. Please don't mind failing integrations, still didn't figure out how to run this from forks.

Thank you for you work :pray:

Reply:
Updated, thank you both 🙏